src/interactive/accordion.story.jsx

```diff
@@ -51,13 +47,54 @@ const defaultPanels = [
 	<AccordionPanel {...panelThreeProps} />,
 ];
 
+/**
+ * @module ControlledAccordionPanel
+ */
+class ControlledAccordionPanel extends React.Component {
+	constructor(props) {
+		super(props);
+
+		this.togglePanel = this.togglePanel.bind(this);
+
+		this.state = {
+			firstOpen: true,
+		};
+	}
+
+	togglePanel() {
+		this.setState({ firstOpen: !this.state.firstOpen });
+	}
+
+	render() {
+		return (
+			<div className="span--100 padding--all">
+				<button onClick={this.togglePanel}>Toggle the first panel</button>
+
+				<AccordionPanelGroup
+					accordionPanels={[
+						<AccordionPanel
+							{...panelOneProps}
+							isOpen={this.state.firstOpen}
+						/>,
+						<AccordionPanel {...panelTwoProps} />,
+						<AccordionPanel {...panelThreeProps} />,
+					]}
+				/>
+			</div>
+		);
+	}
+}
+
 storiesOf('Accordion', module)
 	.addDecorator(decorateWithBasics)
 	.addWithInfo('default', 'Basic Accordion group', () => (
 		<div className="span--100 padding--all">
 			<AccordionPanelGroup accordionPanels={defaultPanels} />
 		</div>
 	))
+	.addWithInfo('controlled accordion', 'Controlled Accordion group', () => (
```

Comment:
I'd either stash this, or change the name of the story to something like "controlled accordion - broken right now"

Reply:
okay